roles/metalk8s_lvm_vg/examples/multipath_fact.json

```diff
@@ -0,0 +1,11051 @@
+192.168.78.28 | SUCCESS => {
+    "ansible_facts": {
+        "ansible_all_ipv4_addresses": [
+            "192.168.78.28"
+        ],
+        "ansible_all_ipv6_addresses": [
+            "fe80::46a8:42ff:fe49:dc79",
+            "fe80::46a8:42ff:fe49:dc79",
+            "fe80::46a8:42ff:fe49:dc79"
+        ],
+        "ansible_apparmor": {
+            "status": "disabled"
+        },
+        "ansible_architecture": "x86_64",
+        "ansible_bios_date": "06/03/2015",
+        "ansible_bios_version": "1.3.6",
+        "ansible_bond0": {
+            "active": true,
+            "device": "bond0",
+            "features": {
+                "fcoe_mtu": "off [fixed]",
+                "generic_receive_offload": "on",
+                "generic_segmentation_offload": "on",
+                "highdma": "on",
+                "large_receive_offload": "on",
+                "loopback": "off [fixed]",
+                "netns_local": "off [fixed]",
+                "ntuple_filters": "off [fixed]",
+                "receive_hashing": "off [fixed]",
+                "rx_checksumming": "off [fixed]",
+                "rx_vlan_filter": "on",
+                "rx_vlan_offload": "on",
+                "scatter_gather": "on",
+                "tcp_segmentation_offload": "on",
+                "tx_checksum_fcoe_crc": "off [fixed]",
+                "tx_checksum_ip_generic": "on",
+                "tx_checksum_ipv4": "off [fixed]",
+                "tx_checksum_ipv6": "off [fixed]",
+                "tx_checksum_sctp": "off [fixed]",
+                "tx_checksum_unneeded": "off [fixed]",
+                "tx_checksumming": "on",
+                "tx_fcoe_segmentation": "off [fixed]",
+                "tx_gre_segmentation": "off [fixed]",
+                "tx_gso_robust": "off [fixed]",
+                "tx_lockless": "on [fixed]",
+                "tx_scatter_gather": "on",
+                "tx_scatter_gather_fraglist": "off [requested on]",
+                "tx_tcp6_segmentation": "on",
+                "tx_tcp_ecn_segmentation": "on",
+                "tx_tcp_segmentation": "on",
+                "tx_udp_tnl_segmentation": "off [fixed]",
+                "tx_vlan_offload": "on",
+                "udp_fragmentation_offload": "off [fixed]",
+                "vlan_challenged": "off [fixed]"
+            },
+            "hw_timestamp_filters": [],
+            "ipv4": {
+                "address": "192.168.78.28",
```

Comment:
You may want to change of the value it give a lot of information publicly (it's already public now...but yet...)

Reply:
I'm personally not worried by a class C ip address but I might be wrong...
Any other thoughts on this ?